src/sentry/static/sentry/app/components/broadcastModal.jsx

```diff
@@ -116,7 +116,7 @@ const BroadcastModal = createReactClass({
         className={'modal ' + nth}
         style={{display: 'block'}}
         key={i}
-        onClick={this.handleClick}
+        onClick={evt => this.handleClick(evt)}
```

Comment:
Shouldn't this be correctly bound already since `BroadcastModal` is using `createReactClass`?

Reply:
Hmm, good point